python/composio/client/enums/base.py

```diff
@@ -37,12 +37,15 @@ class EnumStringNotFound(ComposioSDKError):
 
```

Comment:
Consider adding a docstring to the `EnumStringNotFound` class to improve code documentation. This would help future maintainers understand the purpose and behavior of this exception class. Example:
```python
"""Exception raised when an enum string value is not found.

This exception provides helpful error messages including:
- The invalid value that was provided
- The enum class where the error occurred
- Possible similar valid values (using fuzzy matching)
- Instructions for updating local apps if needed

Args:
    value (str): The invalid enum value that was provided
    enum (str): The name of the enum class
    possible_values (List[str]): List of valid enum values
"""
```